src/NodeJS/Server.php

```diff
@@ -109,9 +109,15 @@ public function __destruct()
      * Setter host
      *
      * @param string $host The server host
+     *
+     * @throws \RuntimeException When server is already running.
      */
     public function setHost($host)
     {
+        if ($this->isRunning()) {
+            throw new \RuntimeException('Unable to change host of a running server.');
```

Comment:
I think a ``LogicException`` would make more sense (it is an error in the logic of your program if you try to do it)